### DISCUSSION TITLE
Product Roadmap 🚀

### DISCUSSION THREAD

**Discussion by marco-souza** [podcodar/app#85] [Announcements]
# Roadmap 🛣️

This is our **Product Roadmap**, which will help to document our steps until here, and also plan the future of our application.

## WebApp

- [x] [Task Workflow](https://github.com/podcodar/app/milestone/1)
- [ ] [Production Release](https://github.com/podcodar/app/milestone/3)
- [ ] [Post Release](https://github.com/podcodar/app/milestone/4)
- [ ] Observability
- [ ] UI Consistency

## Admin

- [ ] [Admin WebApp](https://github.com/podcodar/app/milestone/2)

**Comment by vitorschelb** [comment]
Great knowledge of product workflow. Thanks for sharing.

#This is the Way
